assets/customer.css

```diff
@@ -578,14 +584,9 @@
 }
```

Comment:
I think we can also remove the `color: a:hover;` on `.account table td:first-of-type a:hover` since it's already taken care of: 
![](https://screenshot.click/21-08-8r9fo-mmei4.png)